libpod/kube.go

```diff
@@ -625,13 +625,18 @@ func capAddDrop(caps *specs.LinuxCapabilities) (*v1.Capabilities, error) {
 
 // generateKubeSecurityContext generates a securityContext based on the existing container
 func generateKubeSecurityContext(c *Container) (*v1.SecurityContext, error) {
-	priv := c.Privileged()
+	privileged := c.Privileged()
 	ro := c.IsReadOnly()
 	allowPrivEscalation := !c.config.Spec.Process.NoNewPrivileges
 
-	newCaps, err := capAddDrop(c.config.Spec.Process.Capabilities)
-	if err != nil {
-		return nil, err
+	var capabilities *v1.Capabilities
```

Comment:
Shouldn't this be initialized? (I'm not a Gohead; I may be missing something obvious)

Reply:
That's no required here. Setting the container to privileged implies that all caps get set later on:
```
$ podman (fix-8897) $ podman top 5e5eb6e0fb43 capeff
EFFECTIVE CAPS
full
```